kolena/_experimental/object_detection/dataset.py

```diff
@@ -212,6 +213,7 @@ def _compute_metrics(
     *,
     ground_truth: str,
     inference: str,
+    ignore_gt_property: Optional[str] = None,
```

Comment:
Is there a reason to make this configurable, rather than just using `ignore`? Wonder if it makes the API needlessly verbose.

Reply:
Transparency would be the biggest argument in favor IMO. This makes it clear to the user what field is being used. I could easily see a situation where user uses the name `ignored` instead of `ignore`, for example. In that case, hard-coding the magic property `ignore` would make it pretty difficult to discover the issue.

Reply:
We could alternatively have an `options: Optional[Dict]` argument or similar. I actually like that better than what I have here, come to think of it.